### ISSUE TITLE
linter-gcc does not work

### ISSUE THREAD

**Issue by bitwing** [hebaishi/linter-gcc#41]
Hello, I have installed linter, linter-gcc but it's not work for .c file. linter-clang is worked.
```
gcc --version 
gcc (GCC) 5.3.0
Atom version is 1.6.0
linter-gcc version is 0.6.8

```
Console message:
```
linter-gcc: Using configuration page settings
linter-gcc config: {"execPath":"/usr/bin/gcc","gccIncludePaths":" ","gccSuppressWarnings":false,"gccDefaultCFlags":"-c -Wall","gccDefaultCppFlags":"-c -Wall -std=c++11","gccErrorLimit":0}
linter-gcc: /usr/bin/gcc -c -Wall -fmax-errors=0 /home/test/temp2/hex2dd.c
```
I've googled but can't figure out. Any help is much appreciated.

Thanks a lot.

**Comment by hebaishi** [comment]
Could you paste the linter-gcc command into a terminal so we can look at the output?
The command is:
```
/usr/bin/gcc -c -Wall -fmax-errors=0 /home/test/temp2/hex2dd.c
```
I find it very strange that it's not working for you. I have just installed gcc-5.3 on my computer and everything seems to be working normally. Note that you will not get any output unless you have an error in your C source. A simple thing you can do is delete a semicolon - that should trigger an error.

**Comment by hebaishi** [comment]
Hello. Is linter-gcc working for you now?

**Comment by hebaishi** [comment]
Could you please confirm that linter-gcc is working for you? If not, could you please describe the problem you are having?

**Comment by iskatu** [comment]
I'm experiencing the same issue. linter-gcc does not show anything in Atom but running the command manually correctly returns the errors.

```
Atom 1.7.0
gcc 5.3.0
linter 1.11.4
linter-gcc 0.6.9
```

Console messages:

```
linter-gcc: Using configuration page settings
linter-gcc config: {"execPath":"/usr/bin/g++","gccIncludePaths":" ","gccSuppressWarnings":false,"gccDefaultCFlags":"-c -Wall","gccDefaultCppFlags":"-c -Wall -std=c++11","gccErrorLimit":0}
linter-gcc: /usr/bin/g++ -c -Wall -std=c++11 -fmax-errors=0 /home/test/main.cc
```

Output of `/usr/bin/g++ -c -Wall -std=c++11 -fmax-errors=0 /home/test/main.cc`:

```
/home/test/main.cc:1:11: Fehler: »::main« muss »int« zurückgeben
 void main() {
           ^
/home/test/main.cc: In Funktion »int main()«:
/home/test/main.cc:2:3: Fehler: »sda« wurde in diesem Gültigkeitsbereich nicht definiert
   sda
   ^

```

**Comment by keplersj** [comment]
Looks to me like it's a localization issue. Is there a GCC parameter to
output errors in a different locale?

**Comment by keplersj** [comment]
I believe linter-javac had about the same issue a while ago as well.

**Comment by iskatu** [comment]
Yes, this seems to be the cause of the problem. It works when I have `LC_MESSAGES` set to English.
